bin/strata-client/src/rpc_server.rs

```diff
@@ -901,12 +862,12 @@ impl StrataSequencerApiServer for SequencerServerImpl {
 
 pub struct StrataDebugRpcImpl<D> {
     storage: Arc<NodeStorage>,
-    database: Arc<D>,
+    _database: Arc<D>,
```

Comment:
Why not just remove this if not used? Storage will replace this anyways.

Reply:
I must have missed this.  I was removing the uses of it in one commit, then actually removing the fields in the next, since that trickles down to needing to change the init logic.